readme.md

```diff
@@ -7,14 +7,18 @@ This repository contains the defacto-standard eslint configuration used on all N
 Add the config to your project dependencies:
 
 ```bash
-yarn add netsells/eslint-config-netsells eslint@^4.10.0
+// Add the configuration
+yarn add netsells/eslint-config-netsells
 ```
 
 Extend the config in your project. For example, a `.eslintrc` file in your project root:
 
 ```json
 {
-  "extends": "netsells"
+    "extends": "netsells",
+    "plugins": [
```

Comment:
@jdtjenkins can you look into putting the plugins somewhere else, i _think_ you can put them in modules or just in the config itself. This also raises a concern that the vue stuff isn't actually working as the plugin isn't defined 🤔 

Would you mind taking a look?